### DISCUSSION TITLE
Transfer a repository between two organizations.

### DISCUSSION THREAD

**Discussion by cheld** [actions/create-github-app-token#93] [General]
I have two organizations. I have installed the same github app in both organizations. I would like to transfer a repository between the two organizations using app token.

I assume this is not possible with this action, because I can have only permissions for one owner at a time, correct? Is there an alternative to use personal access token?

**Comment by gr2m** [comment]
> I assume this is not possible with this action, because I can have only permissions for one owner at a time, correct

That is correct

> Is there an alternative to use personal access token?

Yes, you can do it using a personal access token given the token has admin scopes in both organizations

**Reply by cheld** [reply]
Thank you very much
